DataFormats/HcalCalibObjects/data/dosemapHB_4TeV.txt

```diff
@@ -0,0 +1,16 @@
+3.53787e-05 2.58515e-05 1.2848e-05 1.2864e-05 9.64444e-06 6.45818e-06 4.79736e-06 3.41728e-06 2.59323e-06 1.58431e-06 1.30303e-06 7.61866e-07 6.44677e-07 4.22318e-07 3.28374e-07 2.97656e-07 1.3907e-07

```

Comment:
@davidlange6 if you can suggest a different satisfactory location for the code + inputs that will stop the continual requests to move to a different package every time it's updated, please go ahead